assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.js

```diff
@@ -64,7 +70,20 @@ function AudienceTilesWidget( { Widget, WidgetNull } ) {
 	);
 
 	if ( ! hasMatchingAudience ) {
-		return <WidgetNull />;
+		return availableAudiencesSynced ? (
+			<NoAudienceBanner />
```

Comment:
This should render the `NoAudienceBannerWidget` component rather than `NoAudiencesBanner`, at the moment the widget is missing the standard white background.

![image](https://github.com/user-attachments/assets/2588e7b8-5987-4a9f-a53c-28e42d2f2d5f)


```suggestion
			<NoAudienceBannerWidget
				Widget={ Widget }
				WidgetNull={ WidgetNull }
			/>
```